src/analytics/src/main/scala/osmesa/analytics/oneoffs/ChangesetORCCreator.scala

```diff
@@ -0,0 +1,175 @@
+package osmesa.analytics.oneoffs
```

Comment:
@mojodna This is a separate utility for updating our changeset ORC files from the changeset stream.  This automatically finds the sequence number to start changeset stream from.  It seems like this is an improvement from the current state of affairs, but reasonable people can disagree.  You have any thoughts on this?

Reply:
This looks very useful! So much so that maybe it belongs in VP with the other core OSM-related functionality?

I think the name could be clearer. Something along the lines of "merge subsequent changesets".

I also wonder if it makes sense to make the ORC argument optional and allow either sequence numbers (if one has those handy for some reason) or timestamps instead. It would also be nice to include the start/end sequences in the output filename (since there's no easily-accessible metadata otherwise), but in a way that can be overridden.

Reply:
This comment made less sense to me before I realized that I forgot to union the existing changeset ORC to the streamed input.  That's fixed.  Do you still hath concerns about this?

I think what you really want by this comment will mostly be satisfied by bringing the `ChangesetORCUpdaterUtils` into VP's `ChangesetSource`?  Is that somewhat accurate?

Reply:
Largely, yes, but I'm also quibbling about the name of this command. May I propose `MergeChangesets` instead of `ChangesetORCCreator`?

Reply:
I did opt for MergeChangesets, FYI.